### PR TITLE
Nukes Redundant Reachability instance

### PR DESCRIPTION
**Pull request by jleandroperez**
/cc @diegoreymendez 

Last PR of the day, i promise!. Thanks Diego!

### CODE REVIEW

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 1]
```diff
@@ -868,15 +866,9 @@ - (void)setupReachability
 {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Warc-retain-cycles"
```
Comment (diegoreymendez) [inline WordPress/Classes/System/WordPressAppDelegate.m]: This is not part of your PR but it caught my attention... feel like removing this in your PR?  See my comment below...

**Comment by jleandroperez** [comment]
@diegoreymendez Updated, thanks!

**Comment by diegoreymendez** [comment]
One of my favorites PRs :)  :shipit:

**Comment by jleandroperez** [comment]
Thanks for the review Diego!

**Pull request merged**